telethon_examples/quart_login.py

```diff
@@ -1,10 +1,38 @@
 import base64
 import os
 
-from quart import Quart, request
+from quart import Quart, render_template_string, request, session
```

Comment:
Heh, I guess `render_template_string` is better than my hand-rolled `html(inner)` function.